lib/agent/ocsp_response_cache.js

```diff
@@ -59,6 +59,7 @@ function OcspResponseCache()
 {
   let downloadStatus = status.NOT_START;
   let cacheUpdated = false;
+  let cacheInitialized = false;
```

Comment:
Added a flag to indicate whether the cache is created, since the initialization is moved, it's possible the cache does not exist yet when OcspResponseCache is called.